src/mito2/src/memtable/time_series.rs

```diff
@@ -147,7 +147,8 @@ impl TimeSeriesMemtable {
     fn update_stats(&self, stats: WriteMetrics) {
         self.alloc_tracker
             .on_allocation(stats.key_bytes + stats.value_bytes);
-        stats.update_timestamp_range(&self.max_timestamp, &self.min_timestamp);
+        self.max_timestamp.fetch_max(stats.max_ts, Ordering::SeqCst);
+        self.min_timestamp.fetch_min(stats.min_ts, Ordering::SeqCst);
```

Comment:
does the loading part in `Memtable.stats()` be Ordering::SeqCst too? it's Ordering::Relaxed now

Reply:
> does the loading part in `Memtable.stats()` be Ordering::SeqCst too? it's Ordering::Relaxed now

Actually I think here can be also Relaxed too.